packages/x-charts/src/internals/defaultizeMargin.ts

```diff
@@ -0,0 +1,33 @@
+import type { LayoutConfig } from '../models';
+import type { ChartMargin } from './plugins/corePlugins/useChartDimensions/useChartDimensions.types';
+
+export function defaultizeMargin(
+  input: LayoutConfig['margin'],
+  defaultMargin: ChartMargin,
+): ChartMargin;
+export function defaultizeMargin(
+  input: LayoutConfig['margin'],
+  defaultMargin?: ChartMargin,
+): Partial<ChartMargin> | undefined;
+export function defaultizeMargin(
```

Comment:
Although used across our codebase, "defaultize" [doesn't seem to be used](https://dictionary.cambridge.org/spellcheck/english/?q=defaultize). I wonder if we should rename this to `defaultMargin`, `withDefaultMargin` or `withFallbackMargin`. `defaultMargin` would make more sense, but "default" can be a verb or a noun, so it could be confusing. 

Not a blocker, just something to think about.

Reply:
Don't review too much this PR, as it is still in `Draft` 😆

Reply:
But yes, I do agree the `defaultize` used throughout the codebase is weird. It was here before me, I'm just keeping the pattern 😆

You can open an issue to fix that though